### DISCUSSION TITLE
MSI GS75 Stealth 9SG - Wrong Image Format 11!

### DISCUSSION THREAD

**Discussion by WolfJMZ** [xCuri0/ReBarUEFI#185] [General]
Hi there!

I'm trying to get resizable bar enabled for my laptop - which is an MSI GS75 Stealth 9SG.

Has an i7-9750H, Intel HD Graphics 630, and NVIDIA RTX 2080 Max-Q. Chipset is HM370.

I can get to the step of downloading and modifying the UEFI BIOS image from the website, but when I try to perform the BIOS upgrade, I get an error saying "BIOS Update Failed: Wrong Image Format 11!".

The UEFI Patching tool showed "no patches available" so I was able to skip straight to the flashing.

I can see resizable BAR is enabled for the intel integrated graphics, but I want to enable it for the NVIDIA graphics.

There are hidden BIOS settings - where I hold ALT, Right Control, Right Shift, and press F2, I can get access to settings like Memory Remap (enabled) and Aperture Size (256MB). I changed the Aperture Size to 2048MB, but that change isn't reflected and I still see just 256MB.

Let me know what else I can try or do.

This is similar to https://github.com/xCuri0/ReBarUEFI/discussions/155 except I can't actually flash the modded UEFI BIOS I made.

I've attached the original bios and the modded bios I made to this post for reference in case it's an easy fix. I don't have any hardware reflashers nor am I experienced in that realm.

RTX 2080 Max-Q:
![image](https://github.com/xCuri0/ReBarUEFI/assets/8965289/9cb5934e-9df9-4a4d-9da1-e56970f482dd)

Intel 630:
![image](https://github.com/xCuri0/ReBarUEFI/assets/8965289/f727af01-c80d-4fa1-8008-fae0fa49e0ee)

[E17G1IMS.111.zip](https://github.com/xCuri0/ReBarUEFI/files/14584538/E17G1IMS.111.zip)
[E17G1IMS.111-MODDED.zip](https://github.com/xCuri0/ReBarUEFI/files/14584556/E17G1IMS.111-MODDED.zip)

**Comment by MrAnonymous7122** [comment]
Wrong image format probably means you didn't save the file in the correct format after editing.

**Comment by WolfJMZ** [comment]
I heard from SVET at MSI forums. MSI laptops cannot have their BIOS reflashed without being signed by MSI. It just won't work. And using intel's flashing tool to bypass the built-in flashing mechanism won't work either since the BIOS signature is checked at startup.

**Comment by MrAnonymous7122** [comment]
I haven't heard of the bios signature being checked upon each startup before. If that is the case then it's not possible but I would have thought the use of third party flashing software or a physical chip programmer would have worked.

**Comment by WolfJMZ** [comment]
According to SVET - it's because it's a laptop and MSI has it locked down pretty good.

**Reply by MrAnonymous7122** [reply]
It makes sense, a laptop is likely to have more restricting OEM parts than a PC.

**Comment by MrAnonymous7122** [comment]
Have you tried using third party software or MSI has just told you not to?

**Reply by WolfJMZ** [reply]
From what SVET told me, it's not possible even with third party software (such as the intel fpt tools or amibcap)

**Reply by MrAnonymous7122** [reply]
"from what MSI have told you" MSI wouldn't want you trying to do it, but it could be possible. All I can say is try at your own risk but I think it could work.

**Reply by WolfJMZ** [reply]
no - It wasn't MSI that I talked to - it was a user by the name of SVET on the MSI forums via email in the "unofficial BIOS and modding" section. SVET does custom BIOSes and other mods for MSI hardware.

**Reply by MrAnonymous7122** [reply]
Oh sorry, my mistake. I would assume SVET knows what he's talking about so I guess you are out of luck.
